apps/vaporgui/VPushButton.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
```

Comment:
What problem will arise if this pragma is gone?

Reply:
The same question applies to many occurance of this pragma.

Reply:
John's comment summarizes what this is doing nicely.  Maybe this requires more discussion as a group.